### PR TITLE
Add proc creating alerts

### PR DESCRIPTION
**Pull request by mrunge**
add documentation on creating alerts

### CODE REVIEW

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert.adoc [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+// The `context` attribute enables module reuse. Every module's ID includes
+// {context}, which ensures that the module has a unique ID even if it is
+// reused multiple times in a guide.
+//
+// In the title, include nouns that are used in the body text. This helps
+// readers and search engines find information quickly.
+// Do not start the title with a verb. See also _Wording of headings_
+// in _The IBM Style Guide_.
+
+= Creating an alert
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_creating-an-alert.adoc]: Hi Matthias, this is the first information about alerts in our docs. Can you point me to where I can read more about them? We will most like have to include some context here for the user, basically, why create an alert, why use alertmanger.

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert.adoc [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+executing oc create -f <file name> will create a PrometheusRule object
+named prometheus-example-rule-1, which will result in a rule in
+Prometheus.
+
+The same can also be achieved by navigating in the OpenShift web UI to
+the "Installed Operators" menu item under Operators, then click on
+Prometheus Operator followed by clicking on "Prometheus Rule" in
+the sub-menu.
+
+image::prom-operator-screenshot.png[]
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_creating-an-alert.adoc]: No need to use these screenshots here. Your instructions are clear enough.

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert.adoc [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+and finally filling in the empty spots in the next editor window
+
+image::prom-create-rule-2.png[]
+
+For more info in alerting, see link:https://github.com/coreos/prometheus-operator/blob/master/Documentation/user-guides/alerting.md
+
+
+= Creating an alert route in alertmanager
+
+The alertmanager configuration is currently handled as a secret in
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_creating-an-alert.adoc]: When I edit this file in another branch, I will include a x-ref here. I will also probably delete the screenshot.

**Comment by mrunge** [comment]
Included in https://github.com/infrawatch/documentation/commit/fcc7fc3a095e028d6be0984025f995912b1a33ef

**Pull request closed without merging**